### ISSUE TITLE
Publii Integration?

### ISSUE THREAD

**Issue by chrisgliddon** [gmarland/lohost#2]
Publishing from local CMS Publii to lohost seems logical. Then you could have an entirely local workflow, especially for non-developers to use the solutions. Food for thought!

**Comment by gmarland** [comment]
Definitely! That's a really great idea. Currently it relies on knowing html and if it integrated with something like Publii it would open it up completely. Thanks again!
